src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -143,16 +145,25 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
       template = <div>Unknown Template: ${(venue as AnyVenue).template}</div>;
   }
 
-  // @debt remove backButton from Navbar
+  const parentVenueId = venue?.parentId ?? "";
```

Comment:
`venue` will also be defined here. And we don't want to 'hide' non-existent properties by defaulting to empty string:

```suggestion
  const parentVenueId = venue.parentId;
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1492/commits/516bcef90616e538cbf0a8609a96d134bfbf4fcc